front/lib/resources/retrieval_document_resource.ts

```diff
@@ -172,6 +172,15 @@ export class RetrievalDocumentResource extends BaseResource<RetrievalDocument> {
       return null;
     }
 
+    // Prevents users from accessing the document contents of a public data source
```

Comment:
Why?

Reply:
During testing, I realized we might not want to redirect users to the public data source view when they click on documents used in an agent message. I’ve temporarily disabled this, but we can always revisit it later.